agent/main.go

```diff
@@ -126,5 +119,8 @@ func main() {
 		log.G(ctx).WithError(err).Warn("shim error")
 	}
 
-	log.G(ctx).Info("done")
+	log.G(ctx).Info("shutting down runc")
```

Comment:
What do you mean by `shim` here? agent or runc shim?
I would rather not use word `shim` inside firecracker at all to avoid confusion with our runtime shim

Reply:
runc isn't running right? So I think it might be confusing to talk about shutting down runc?

Reply:
Next line of code looks as `runcTaskService.Shutdown`, but I don't feel strongly about it.
Do you want something like `shutting down agent` instead?

Reply:
Yes I think that would be better. Someone looking at the logs isn't going to know what the next line of code is.

Reply:
Done